FrontEnd/src/views/AddNgoView.vue

```diff
@@ -142,10 +161,60 @@ let newNgo = reactive({
   location: "",
   pincode: "",
 });
+
 const store = useNgoStore();
 
+let msg = reactive([]);
+
 const postNgo = async () => {
-  await store.postNgo(newNgo);
-  return router.push("/ngos");
+  let validationRegex = {
```

Comment:
a function shouldn't do more than one job here our intention is to POST so let us move out this validation to a different utility function in a separate Javascript file called "helper.js" under "src" directory. So that we can also reuse it in other places easily.